Meme-ify-Me/app/src/main/res/layout/activity_vanilla_meme_edit.xml

```diff
@@ -0,0 +1,161 @@
+    android:layout_height="match_parent"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    tools:context="nyc.c4q.rosmaryfc.meme_ify_me.VanillaMemeEdit">
+
+    <!-- meme preview -->
+    <LinearLayout
+        android:id="@+id/meme_preview"
+        android:layout_weight="2"
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@drawable/successkid">
+
+        <TextView
+            android:id="@+id/top_textView"
+            android:layout_weight="1"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:gravity="center"
+            android:text="top" />
+
+        <TextView
+            android:id="@+id/mid_textView"
+            android:layout_weight="1"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:gravity="center"
+            android:text="middle" />
+
+        <TextView
+            android:id="@+id/bottom_textView"
+            android:layout_weight="1"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:gravity="center"
+            android:text="bottom" />
+
+    </LinearLayout>
+
+    <!-- editTexts and buttons -->
+    <LinearLayout
+        android:id="@+id/editTexts_and_buttons"
+        android:orientation="vertical"
+        android:layout_weight="4"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <!--top, mid, bottom edit texts -->
+        <LinearLayout
+            android:id="@+id/all_editTexts"
+            android:orientation="vertical"
+            android:layout_weight="1"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+
+            <!--top edit text and preview button -->
+            <LinearLayout
+                android:orientation="horizontal"
+                android:layout_weight="1"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <EditText
+                    android:id="@+id/top_editText"
+                    android:layout_weight="1"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:hint="top text" />
+
+                <Button
+                    android:id="@+id/top_editText_preview_btn"
+                    android:layout_weight="2"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:text="Preview" />
+
```

Comment:
Maybe we can make the preview button on the layout 1 instead of 3 and take all 3 textEdit input at once.  If there is any empty line, it will be empty anyways.  What do you think?